.github/workflows/build-ci-buildbox-images.yaml

```diff
@@ -0,0 +1,82 @@
+name: Build CI Buildbox Images
+run-name: Build CI Buildbox Images
+on:
+  push:
+    paths:
+      - build.assets/Dockerfile
+      - build.assets/Dockerfile-centos7
+      - build.assets/Makefile
+      - build.assets/images.mk
+    branches:
+      - master
```

Comment:
What about the release branches? Sometimes we have to update the buildboxes for those too.

Reply:
You mean, when a tag is bumped? I guess, we could either add 'release/**' pattern here or remove branch constraint for `pull_request`. Done the latest for now, do not see clear pattern in release branches. Just keep in mind that we have a single target image everywhere. So, if there would be two concurrent PR's, which change buildbox with a same tag, it could be the problem. My original intention was to build images from master only.